content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -77,7 +77,7 @@ A [Probe](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#prob
 performed periodically by the [kubelet](/docs/admin/kubelet/)
 on a Container. To perform a diagnostic,
 the kubelet calls a
-[Handler](https://godoc.org/k8s.io/kubernetes/pkg/api/v1#Handler) implemented by
+[Handler](https://godoc.org/k8s.io/api/core/v1#Handler) implemented by
```

Comment:
I recommend:
```suggestion
[Handler](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#handler-v1-core) implemented by
```

Does that make sense?

Reply:
Definitely. Using the generated reference should be the better way.